packages/ui-components/src/components/tag/test/tag.e2e.ts

```diff
@@ -0,0 +1,18 @@
+import { E2EPage, newE2EPage } from '@stencil/core/testing';
+
+describe('Tag (end-to-end)', () => {
+	let page: E2EPage;
+
+	describe('when it renders with text', () => {
+		beforeEach(async () => {
+			page = await newE2EPage();
+			await page.setContent('<kv-tag text="Default Text"></kv-tag>');
+		});
+
+		it('should render a tag with the text Default Text', async () => {
```

Comment:
Will it make sense to add a test to when the tag has a slot?

Reply:
Added test bellow

Reply:
and if label was optional  you should add another test without label